Vagrantfile

```diff
@@ -193,7 +193,15 @@ def provision(config,
   raise ArgumentError.new('update_command is required') if update_command == 'required'
   raise ArgumentError.new('update_tracking_file is required') if update_tracking_file == 'required'
   raise ArgumentError.new('install_command is required') if install_command == 'required'
-  config.vm.provision "bats dependencies", type: "shell", inline: <<-SHELL
+  config.vm.provider "virtualbox" do |v|
+    # Give the box more memory and cpu because our tests are beasts!
+    v.memory = Integer(ENV['VAGRANT_MEMORY'] || 8192)
+    v.cpus = Integer(ENV['VAGRANT_CPUS'] || 4)
+  end
+  config.vm.synced_folder "#{Dir.home}/.vagrant/gradle/caches", "/home/vagrant/.gradle/caches",
```

Comment:
IIRC, gradle opens a lock somewhere under the caches dir. If we use the same directory that the parent gradle process (which is calling vagrant) uses, then the inner build fails because the lock is held.